radis/lbl/base.py

```diff
@@ -665,17 +665,29 @@ def get_Evib_CDSD_pcN_1iso(df, iso):
             # the map below is crazy fast compared to above loop
             df["Evibu"] = df_pcN.index.map(Evib_dict.get).values
 
-            return df.loc[idx, ["Evibl", "Evibu"]]
+            try:
+                return df.loc[idx, ["Evibl", "Evibu"]]
+            except:
+                return df.loc[df.index.tolist(), ["Evibl", "Evibu"]]
```

Comment:
@gagan-aryan  why did you need this change ?

Reply:
I had to change this because I created an if-else block in L677 - L690. So idx was no longer accessible when `get_Evib_CDSD_pcN_1iso` was called from the else block.

Reply:
Following up : this `tolist()` is unusual to me; but I now understand why you had to add it. 

It should be a simpler syntax : 
```
return df.loc[:, ["Evibl", "Evibu"]]
```
on the model of `get_Evib123_RADIS_cls5_1iso` (L1149). `idx` here is undefined ; it seems like an error, which didn't have consequence because the "idx" would be picked from the parent function, and correspond to the same lines. 

The syntax above will fix everything.

Reply:
There is the same problem L555. You cna fix it